build.gradle

```diff
@@ -55,7 +55,7 @@ dependencies {
     final javadocJDKFiles = files(((URLClassLoader) ToolProvider.getSystemToolClassLoader()).getURLs())
     compileOnly(javadocJDKFiles)
     testCompile(javadocJDKFiles)
-    compile 'org.freemarker:freemarker:2.3.18'
+    compile 'org.freemarker:freemarker:2.3.23'
```

Comment:
Ok, sounds good. I had tried running the same test last night so, this seems good to go.